pshtt.py

```diff
@@ -116,7 +117,7 @@ def ping(url, allow_redirects=False, verify=True):
         verify=verify,
 
         # set by --user_agent
-        data={'User-Agent': USER_AGENT},
+        headers={'User-Agent': USER_AGENT},
```

Comment:
This fixes a bug where the user agent was not actually being sent. The `data` parameter submits form data, not custom headers. This was causing us to submit the `User-Agent` string as a form data submission, which also added headers to the request and provided data that's not usually present on a `GET` request. While most servers ignored it, this caused Google's front-end servers to drop the traffic and return an error (instead of a redirect), which is how I noticed this issue.